### PR TITLE
MAINT: Refactor code in views for readability

### PR DESCRIPTION
**Pull request by freddiev4**

### CODE REVIEW

**Comment by freddiev4** [comment]
@csuksangium can you take a look at this?

**Comment by freddiev4** [comment]
Let me know if there's any other cleaning up we should do or refactors you have questions about

File: app/views.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # django framework packages
+from django.db import IntegrityError
```
Comment (csuksangium) [inline app/views.py]: not sure if you still need this if handled IntegrityError through models.py

File: app/views.py [hunk 2]
```diff
@@ -264,6 +281,8 @@ def get_facebook_info(user):
             if k == 'hometown':
                 user_info.hometown = fb_user_info['hometown']['name']
             if k == 'friends':
+                import pdb
```
Comment (csuksangium) [inline app/views.py]: pdb is still here from debugging

**Comment by freddiev4** [comment]
@csuksangium updated those lines. Good for another review

**Review by csuksangium — APPROVED** [review]
everything looks good

**Pull request merged**